sql/driver/wire.go

```diff
@@ -55,6 +55,13 @@ func makeDatum(val driver.Value) (Datum, error) {
 		datum.Payload = &Datum_TimeVal{
 			&timestamp,
 		}
+	case Date:
+		datum.Payload = &Datum_DateVal{
+			&Datum_Timestamp{
```

Comment:
ah, yeah. that form was introduced after this commit was originally written. Fixed.